
FAIL Verify a blob URL created via URL.createObjectURL from the third-party context shouldn't be useable as the shared worker script when it's passed to saa_handle.SharedWorker. assert_equals: expected "Blob URL SharedWorker tests completed successfully." but got "Unable to load handle in same-origin context for BlobURLSharedWorker"

